compass_vue/helpers/utils.js

```diff
@@ -16,4 +16,9 @@ function getYesterday() {
   return tomorrow.format("MMMM D, YYYY");
 }
 
-export { formatDate, getToday, getYesterday };
+function getUserRole() {
+  // mock a user role for dev and testing
+  return { role: "Super" };
```

Comment:
Yeah I think this is where we would do map the local "Role" class variable to the role defined in the server's response context